apps/website/blog/2024-05-08-ethdam.md

```diff
@@ -8,6 +8,7 @@ authors:
   url: https://x.com/ctrlc03
   image_url: https://avatars.githubusercontent.com/u/93448202?v=4
 tags: [security, bug, babyjubjub, input validation]
+excerpt: "During [ETHDam's Quadratic Funding round](https://ethdamqf.com/#/), run on [clr.fund](https://clr.fund), we discovered a critical bug in MACI. The issue stemmed from the lack of validation on MACI public keys within the [Poll contract](/docs/developers-references/smart-contracts/solidity-docs/Poll). A user (spoiler alert, it was a self-inflicted denial of service (DoS)) was able to submit a MACI public key which was not a point of the Baby JubJub elliptic curve, and it broke everything."
```

Comment:
given the markdown is not rendering in the card preview, can remove links

Reply:
done